tests/integration/devfile/cmd_devfile_watch_test.go

```diff
@@ -165,6 +165,42 @@ var _ = Describe("odo devfile watch command tests", func() {
 
 	})
 
+	Context("when executing odo watch after odo push with debug flag", func() {
+		It("should be able to start a debug session after push with debug flag using odo watch and revert back after normal push", func() {
+			helper.CmdShouldPass("odo", "create", "nodejs", "--project", commonVar.Project, cmpName)
+
+			helper.CopyExample(filepath.Join("source", "devfiles", "nodejs", "project"), commonVar.Context)
+			helper.CopyExampleDevFile(filepath.Join("source", "devfiles", "nodejs", "devfile-with-debugrun.yaml"), filepath.Join(commonVar.Context, "devfile.yaml"))
```

Comment:
if you run `odo create` after this you will save request on devfile registry.
`odo create` can work with the devfile if it is already present in the current directory

Reply:
done